### ISSUE TITLE
Android import BuildConfig and R

### ISSUE THREAD

**Issue by astulip** [react-native-community/cli#438]
Should applicationId be changed to packageName
My configuration is as follows
Manifest packageName: com.noohle
Gradle applicationId: com.noohle.wk.erp

But BuildConfig and R are packageName information.

https://github.com/react-native-community/cli/pull/435/commits/e46b9687582e17292d76ee69bc3b036226ba5692

**Comment by thymikee** [comment]
Cc @robertying

**Comment by robertying** [comment]
Hey @astulip, thanks for pointing out the problem. I tested it and you were right! Awesome job 😊

So for my commit, I just added extra `import *.R`, totally unware of the problem since `import *.BuildConfig` was already using `applicationId`.

Now that you brought that up, I did some study and found `R` and `BuildConfig` are located using `packageName` rather than `applicationId`.

I figured out a way to get `packageName` from `AndroidManifest.xml` and created a pull request #439. Hopefully this will solve the issue.
